stories/pagination/pagination.stories.mdx

```diff
@@ -90,6 +90,45 @@ The `showFirstAndLastNav` prop allows the user to jump directly to the first and
     </Story>
 </Canvas>
 
+<Heading3>With Page Size Changer</Heading3>
```

Comment:
story wise can you separate this into

1. `With page size changer` (note casing) that uses the default options
2. `With custom page size options` that uses the custom page options